x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/aggregate/MaxTests.java

```diff
@@ -49,73 +49,6 @@ public static Iterable<Object[]> parameters() {
 
         suppliers.addAll(
             List.of(
-                // Surrogates
```

Comment:
These tests were copied from Top(). But they don't make sense here, they don't add any special case. So I'm removing to simplify things a bit